cpu.go

```diff
@@ -137,7 +145,49 @@ func (tsa *TimeSeriesAverage) Percentage() (map[int]ValuesMap, error) {
 				sum[d][key] = -1
 				continue
 			}
-			sum[d][key] = float64(int64(((lastVal-tsa.TimeSeries[keyInt].Values[key])/last.Time.Sub(tsa.TimeSeries[keyInt].Time).Seconds())*10000+0.5)) / 100
+			// On windows perCPU measurements returned as Percentage, not CPU time
+			// see https://github.com/shirou/gopsutil/issues/600
+			if runtime.GOOS == "windows" {
+				var valSum float64
+				var count int
+				for i := keyInt; i < len(tsa.TimeSeries); i++ {
+					t := tsa.TimeSeries[i]
+					// skip measurements collected more than d minutes ago (e.g. 5 minutes for avg5)
+					if time.Since(t.Time).Minutes() > float64(d) {
+						continue
+					}
+
+					valSum += t.Values[key]
+					count++
+				}
+
+				// found the average for all collected measurements for the last d minutes
+				sum[d][key] = roundUpWithPrecision(valSum/float64(count), 2)
+
+			} else {
+
+				var hasMetrics bool
+				// filter out measurements collected more than d minutes ago (e.g. in case some of them were timeouted)
+				for i := keyInt; i < len(tsa.TimeSeries); i++ {
+					// allow 3 seconds(0.05min) outage to include metrics query time
+					if time.Since(tsa.TimeSeries[i].Time).Minutes() <= float64(d)+0.05 {
+						keyInt = i
+						hasMetrics = true
+						break
+					}
+				}
+				if !hasMetrics || keyInt == len(tsa.TimeSeries)-1 {
+					// looks like some problem happen and we don't have enough(more than 1) measurements
```

Comment:
Try to keep it consistent. Don't start one line with a small letter and the next one with a capital one